misc/anonymization.md

```diff
@@ -92,3 +92,34 @@ Computed Properties:
 
 
 <sub>*\* Features that are marked ~~strikethrough~~ are not implemented yet.*</sub>
+
+
+## Anonymization
+You can manually anonymize `Attachment` using management command:
+
+```
```

Comment:
@martinmacko47 Dobre sa whitespacesuje ak je to pisane ako kod. Nechat aj Options ktore nie su nami definovane?

Reply:
@viliambalaz V dokumentacii popisujeme iba nase parametre. Parametre pochadzajuce z Djanga su odokumentovane v Djangu. Toto nie je uzivatelska prirucka, ale technicka dokumentacia pre nas.

Reply:
@viliambalaz 
> Dobre sa whitespacesuje ak je to pisane ako kod.

Co tak odrazky? Akoze copy&paste vystupu commandu je ozaj super dokumentacia. 🤦‍♂️